xtask/src/extendrtests/with_absolute_path.rs

```diff
@@ -41,31 +42,57 @@ pub(crate) fn swap_extendr_api_path(shell: &Shell) -> Result<DocumentHandle, Box
 
     let mut replacement = InlineTable::new();
 
-    let item = Value::from(get_replacement_path(&current_path));
+    let item = Value::from(get_replacement_path_extendr_api(&current_path));
     replacement.entry("path").or_insert(item);
     *extendr_api_entry = Value::InlineTable(replacement);
 
+    #[allow(non_snake_case)]
+    let libR_sys_entry =
+        get_libR_sys_entry(&mut cargo_toml).ok_or("`libR-sys` not found in Cargo.toml")?;
+
+    let mut replacement = InlineTable::new();
+
+    let item = Value::from(get_replacement_path_libR_sys(&current_path));
+    replacement.entry("path").or_insert(item);
+    *libR_sys_entry = Value::InlineTable(replacement);
+
     shell.write_file(CARGO_TOML, cargo_toml.to_string())?;
-    Ok(DocumentHandle {
+    Ok(DocumentMutHandle {
         document: original_cargo_toml_bytes,
         shell,
     })
 }
 
-fn get_replacement_path(path: &Path) -> String {
+fn get_replacement_path_extendr_api(path: &Path) -> String {
     let path = path.adjust_for_r();
 
     format!("{path}/extendr-api")
 }
 
-fn get_extendr_api_entry(document: &mut Document) -> Option<&mut Value> {
+#[allow(non_snake_case)]
+fn get_replacement_path_libR_sys(path: &Path) -> String {
```

Comment:
Do we also rely on some adjacent `libR-sys` folder?

Reply:
See comment from before.

Reply:
I guess what I'd like is that for this to work if have a git submodule for `libR-sys` and `rextendr`, and if you *don't*. If you could make that happen, I'd be eternally thankful.

Reply:
What do you think should happen if you do not have submodules? Should we just skip replacement of libR-sys?

Reply:
Yes.... That makes sense. Hmm.... Some of this is done by copying code you wrote..